lightning/src/routing/mod.rs

```diff
@@ -19,7 +19,8 @@ use routing::network_graph::NodeId;
 ///
 ///	Scoring is in terms of fees willing to be paid in order to avoid routing through a channel.
 pub trait Score {
-	/// Returns the fee in msats willing to be paid to avoid routing through the given channel.
+	/// Returns the fee in msats willing to be paid to avoid routing through the given channel from
+	/// `source` to `destination`.
```

Comment:
This isn't the `source` and `dest` of the payment itself, right? Just the forwarder/forwardee of the given `short_channel_id`? Those names are a bit overloaded for me lol